packages/dso-toolkit/src/styles/components/_dropdown.scss

```diff
@@ -154,3 +154,11 @@ $dso-dropdown-hor-padding: 20px;
     }
   }
 }
+
+.text-right {
```

Comment:
In `packages\dso-toolkit\libs\bootstrap\_navbar.scss` staat:
```@media (min-width: $grid-float-breakpoint) {
  .navbar-left {
    float: left !important;
  }
  .navbar-right {
    float: right !important;
  margin-right: -$navbar-padding-horizontal;

    ~ .navbar-right {
      margin-right: 0;
    }
  }
}
```
... en dat is wat er nu op dso-toolkit.nl voor zorgt dat de dropdown daar rechts uitlijnt. 
Moeten we daar nog wat mee?

Reply:
We hebben hier natuurlijk eigenlijk te maken met 2 aspecten: de vormfactor van de dropdown zelf (of het uitklapmenu naar links of naar rechts springt), en de positionering van de dropdown werkvorm als geheel.

Reply:
Dat gezegd hebbende, vind ik `text-right` eigenlijk een dubieuze classname voor deze toepassing. Het gaat toch helemaal niet over de tekst? Het gaat over de uitklaprichting van het menu!

Reply:
`.navbar-right` zorgt ervoor dat die navigatiebalk (dropdown button EN gitbub link) rechts uitlijnt. Met de uitlijning van de dropdown zelf heeft dat niks te maken.